client/quantum_serverless/core/constrants.py

```diff
@@ -21,3 +21,9 @@
 OT_SPAN_DEFAULT_NAME = "entrypoint"
 OT_ATTRIBUTE_PREFIX = "qs"
 OT_LABEL_CALL_LOCATION = "qs.location"
+
+
+# repository
+REPO_HOST_KEY = "REPO_HOST_KEY"
+REPO_PORT_KEY = "REPO_PORT_KEY"
+REPO_PREFIX_KEY = "REPO_PREFIX_KEY"
```

Comment:
I see that `REPO_PREFIX_KEY` is not being used, for what do we need this variable? Maybe we can remove it and use it when we need it.